internal/builders/generic/README.md

```diff
@@ -343,4 +343,5 @@ jobs:
     uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@v1.1.1
     with:
       base64-subjects: "${{ needs.goreleaser.outputs.hashes }}"
-```
\ No newline at end of file
+      upload-to-release: true
```

Comment:
nit: The default is still true and I think the name will change anyway so you can remove this.
```suggestion
```